pom.xml

```diff
@@ -21,19 +21,19 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-  <parent>
-    <artifactId>mojo-parent</artifactId>
-    <groupId>org.codehaus.mojo</groupId>
-    <version>38</version>
-  </parent>
+	<parent>
+		<groupId>io.cotiviti</groupId>
+		<artifactId>cotiviti-parent</artifactId>
+		<version>0.3.5-SNAPSHOT</version>
+	</parent>
```

Comment:
This was inadvertently changed in our fork's `master`.  I will figure it out and re-try.